frontend/dynamic-demo-plugin/i18next-parser.config.js

```diff
@@ -0,0 +1,11 @@
+const FALLBACK_LOCALE = 'en';
```

Comment:
Is this actually needed?

I can't see this option listed in [`i18next-parser` options](https://github.com/i18next/i18next-parser#options) :thinking:

Reply:
This was copied and pasted from here:

https://github.com/spadgett/console/blob/093d3fee0aae7d4af86936fa64f43acfca556d61/frontend/i18next-parser.config.js#L3

But I don't see where it is ever used. That was probably meant to be `fallbackLng`. I'll take it out.

cc @rebeccaalpert